dev-python/pydocstyle/pydocstyle-2.0.0.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python{2_7,3_{4,5,6}} )
+
+inherit distutils-r1
+
+DESCRIPTION="Static analysis tool for checking compliance with Python docstring conventions"
+HOMEPAGE="https://github.com/PyCQA/pydocstyle"
+SRC_URI="https://github.com/PyCQA/pydocstyle/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+RDEPEND=""
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]
+	dev-python/six[${PYTHON_USEDEP}]
+	=dev-python/snowballstemmer-1.2.1[${PYTHON_USEDEP}]
+	test? (
+		virtual/python-pathlib[${PYTHON_USEDEP}]
+		=dev-python/mock-2.0.0[${PYTHON_USEDEP}]
+		dev-python/pytest[${PYTHON_USEDEP}]
+		dev-python/pytest-pep8[${PYTHON_USEDEP}]
+	) "
+
+python_test() {
+	esetup.py test
```

Comment:
```
----------------------------------------------------------------------
Ran 0 tests in 0.001s
```

You still aren't running tests.

Reply:
Did you enable FEATURE="test"? It runs them for me.

Reply:
Yes, I did. That's how I got the `0 tests` output.

Reply:
I'm really stumped by this one. I have it running the tests when I install it. I'll reach out on the mailing list for some help.

Reply:
The upstream-ish way to run them would be via `py.test -v` (which implies pytest dependency). If you're reaching up to them, you may ask them to rewrite them to be compliant to standard Python `unittest` module.